seed.js

```diff
@@ -51,50 +56,90 @@ var seedUsers = function () {
 
 };
 
+
+var seedFilms = function () {
+
+    var films = [
+    {
+        title: 'Die Hard',
+        categories: ['5532c18a0dad5827ab24af3c', '5532c18a0dad5827ab24af49', '5532c18a0dad5827ab24af46'],
```

Comment:
What happens when reseeding and mongo has generated new ID's for the categories?